app/models/manageiq/providers/redhat/connection_manager.rb

```diff
@@ -14,8 +14,6 @@
 # limitations under the License.
 #
 
-require 'ovirtsdk4'
-
 #
 # This class is a responsible for managing a set of `OvirtSDK4::Connection` objects. Each connection is
```

Comment:
Is this comment needed anymore?  Seems like the base class has it covered.

Reply:
:+1: removed